tests/python/pants_test/backend/python/BUILD

```diff
@@ -49,7 +49,7 @@ python_tests(
     'testprojects:pants_plugins_directory',
   ],
   tags={'integration'},
-  timeout = 120,
+  timeout = 360,
```

Comment:
Are these timeouts still intended now that you've improved Pex performance?

Reply:
Oops - I meant to revert those. Reverted.